packages/Molecules/Pagination/index.vue

```diff
@@ -25,6 +33,7 @@ import RplIcon from '@dpc-sdp/ripple-icon'
 export default {
   name: 'RplPagination',
   props: {
+    label: String,
```

Comment:
Would setting a default of 'search' here instead of in the template be better?

Reply:
fixed dat